com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -217,8 +217,12 @@ private void OnValidate()
 #if UNITY_EDITOR
                 UnityEditor.EditorApplication.delayCall += () =>
                 {
-                    UnityEditor.EditorUtility.SetDirty(this);
-                    UnityEditor.SceneManagement.EditorSceneManager.MarkSceneDirty(activeScene);
+                    // 'this' is sometimes invalid by the time the delayed call executes
+                    if (this != null && activeScene != null)
+                    {
+                        UnityEditor.EditorUtility.SetDirty(this);
+                        UnityEditor.SceneManagement.EditorSceneManager.MarkSceneDirty(activeScene);
+                    }
```

Comment:
a bit off-topic but should we also merge this fix to `develop` branch later as well?